src/content/basics/automatic-termination-of-bad-nodes/index.md

```diff
@@ -11,43 +11,40 @@ categories: ["basics"]
 
 ## Overview
 
-Giant Swarm's `terminate-unhealthy-nodes` feature helps you keep the nodes in your cluster in a healthy, running state. When enabled, a service makes periodic checks on the health of each node in your cluster. If a node fails consecutive health checks over an extended time period, it will be drained and terminated.
+Degraded nodes in a Kubernetes cluster should be a rare issue, however when it occurs, it can have severe consequences for the workloads scheduled to the affected nodes. The goal should be to detect bad nodes early and remove them from the cluster, replacing them with healthy ones.
 
-This is an `alpha` feature available on these providers:
+Starting with release v12.6.0 for AWS, you now have the option to automate the detection and termination of bad nodes. When enabled, all nodes in your cluster are periodically checked. If a node fails consecutive health checks over an extended time period, it will be drained and terminated.
 
-* AWS - `12.6.0` release onwards
+This function is available on AWS only currently. Support on other providers will follow.
```

Comment:
dropping the info from the list means you really need to read the whole text and find the release somewhere in the middle of the docs, I consider it super unfriendly for a technical person who needs to quickly find that specific information.

Reply:
I personally often do that I google again some article I already read to find one specific info, being it in a paragraphs makes it harder to see